src/transformState.js

```diff
@@ -5,7 +5,29 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case 'clear':
+        for (const key in state) {
+          delete state[key];
+        }
+        break;
+
+      case 'removeProperties':
+        for (const keyToRemove of action.keysToRemove) {
+          delete state[keyToRemove];
+        }
+        break;
+
+      case 'addProperties':
+        for (const extraKey in action.extraData) {
+          state[extraKey] = action.extraData[extraKey];
+        }
+        break;
+
+      default: ;
```

Comment:
Add some action for a default case 
![image](https://github.com/mate-academy/js_stateful-object/assets/99403198/d0624c60-bade-4562-a315-3ae2b36ac13d)